### ISSUE TITLE
[Icons] - Update documentation and Sketch library to use fa-times-circle

### ISSUE THREAD

**Issue by GnatalieH** [department-of-veterans-affairs/vets-design-system-documentation#996]
## Description
Currently we show fa-times as the close icon [in our documentation](https://design.va.gov/foundation/icons#interaction) and in the Sketch library. However, we actually use fa-times-circle in the developed alert and modal components. We should update the documentation and Sketch library to use fa-times-circle to match our developed components.

## Tasks
- [ ] Update icon documentation on design.va.gov to use fa-times-circle and remove fa-times
- [ ] Update Sketch pattern library to use fa-times-circle and remove fa-times
- [ ] Request documentation draft review
- [ ] Once documentation is approved, publish to design.va.gov

## Acceptance Criteria
- [ ] Icon documentation is published on design.va.gov
- [ ] Sketch library has been updated

**Comment by humancompanion-usds** [comment]
This is superseded by events and is no longer relevant.
